skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -1074,21 +1074,31 @@ espm::Effects::Data espm::Effects::GetData(
   }
 
   Data result;
-  uint32_t effect = 0;
+  uint32_t effectIndex = 0;
+  bool orderFlag = true;
```

Comment:
Something bothers me about this name... but could not come up with something better than something like `lastWasEFID`, `insideRecord`. So I guess `orderFlag` is ok. But maybe this inspired you to a better name